### PR TITLE
[docs] fix more markdown issues and enable more checks

### PR DESCRIPTION
**Pull request by blinkov**
### Changelog category 

* Documentation (changelog entry is not required)

### Additional information

...

### CODE REVIEW

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 1]
```diff
@@ -9,13 +9,14 @@ To initiate the check, call the `SelfCheck` method from `NYdb::NMonitoring` name
...
   App code snippet for creating a client:
+
   ```cpp
   auto client = NYdb::NMonitoring::TMonitoringClient(driver);
   ```
 
   Calling `SelfCheck` method:
 
-  ```
+  ```c++
```
Comment (anton-bobkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:19]:
```suggestion
  ```cpp
```

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md [hunk 2]
```diff
@@ -349,8 +344,8 @@ The status (severity) of the current issue:
 
   - Primary Source: `/sys/fs/cgroup/cpu.max`
   - Fallback Sources: `/sys/fs/cgroup/cpu/cpu.cfs_quota_us`, `/sys/fs/cgroup/cpu/cpu.cfs_period_us`
--
- The number of cores is calculated by dividing the quota by the period (quota / period).
+
+The number of cores is calculated by dividing the quota by the period $(quota / period)$.
```
Comment (anton-bobkov) [inline ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:348]:
```suggestion
The number of cores is calculated by dividing the quota by the period `(quota / period)`.
```

Comment (blinkov) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:348]: @anton-bobkov that was intentional, see https://diplodoc.com/docs/ru/syntax/additional#formulas

File: ydb/docs/en/core/reference/ydb-sdk/health-check-api.md (hunk 1 above)
Comment (blinkov) [reply ydb/docs/en/core/reference/ydb-sdk/health-check-api.md:19]: I think both work the same, but ok

**Review by anton-bobkov — DISMISSED** [review]

**Review by anton-bobkov — APPROVED** [review]

**Pull request merged**